test/models/parser/page_item_test.rb

```diff
@@ -491,21 +470,19 @@ def fake_twitter_user
   end
 
   test "uses twitter URL from twitter metadata for author_url (and not username) if valid" do
-    Twitter::REST::Client.any_instance.stubs(:user).returns(fake_twitter_user)
-
+    # it's getting the author_url from the metadata, but because of how we are now building the url it fails that assert
```

Comment:
let's go ahead and just delete the assertion and comment, rather than commenting! but we should make sure to test the author_url construction somewhere (this may be happening already). can be its own test if that's easiest.

Reply:
I looked through this one again and it's working, we are still testing the author_url construction. 😁